department-of-veterans-affairs/va.gov-cms
# VAGOV-4316: housing migration

To test:
On PR env,
1. Go to /admin/structure/migrate/manage/va_new_benefits/migrations
2. Run 'Migrate Housing benefits pages from Heroku' and 'Migrate Housing assistance sidebar menu'
3. Go to /taxonomy/term/68 ('Housing assistance benefits hub'). You should see 12 draft nodes. Edit one of the nodes. It should have a menu item (in the right rail) with a parent under 'Housing assistance'.
